src/models/entities/Challenge.ts

```diff
@@ -69,9 +71,41 @@ export class Challenge extends BaseEntity implements IChallenge {
   )
   ecoverse?: Ecoverse;
 
+  // The restricted group names at the challenge level
+  @Column('simple-array')
+  restrictedGroupNames?: string[];
+
   constructor(name: string) {
     super();
     this.name = name;
+    this.context = new Context();
+    this.restrictedGroupNames = [RestrictedGroupNames.Members, RestrictedGroupNames.Admins];
```

Comment:
We should redesign our model to support entities with roles and not rely on strings

Reply:
Replace admin group implemenation when we bring in proper authorisation?

Members needed regardless so that we can manage membership - this group is needed, not just for authorisation.